open-company/open-company-web
# Add comment highlight also in the secure post view.

Card: https://trello.com/c/6UG8a63c

---

### Review after #1002 
---

To test:
- copy a link from a comment notification for user A
- also open the same post with user B (with normal jwt login)
- open user A link in incognito
- [x] do you see the comment highlight?
- now add a new comment with user A
- [x] do you see the new comment being highlighted?
- [x] do you see the new comment coming in and being highlighted with user B?
- now add a new comment with user B on the expanded post view
- [x] do you see the comment being highlighted only 1 time?

NB: user A doesn't see the comment from user B coming in live because we don't have interaction connection with the id-token login ATM
